app/formatters/duty_expression_formatter.rb

```diff
@@ -77,14 +81,21 @@ def format(opts = {})
           output << monetary_unit
         end
         if measurement_unit_abbreviation.present?
-          output << if opts[:formatted]
+          output << if formatted
                       "/ <abbr title='#{measurement_unit.description}'>#{measurement_unit_abbreviation}</abbr>"
                     else
                       "/ #{measurement_unit_abbreviation}"
                     end
         end
       end
-      output.join(' ').html_safe
+
+      result = output.join(' ').html_safe
+
+      if resolved_meursing_component
+        "<strong>#{result}</strong>"
```

Comment:
`helper.tag.strong(result)` ?

Reply:
I'm probably going to completely rewrite the duty expression formatter at some point and I'll include that view helper/clarify a bunch of the business logic